turbinia/workers/sshd_test.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for the SSHD analysis task."""
+
+from __future__ import unicode_literals
+
+import unittest
+
+from turbinia import config
+from turbinia.workers import sshd
+
+
+class SSHDAnalysisTaskTest(unittest.TestCase):
+  SSH_INSECURE_EVERYTHING = """PermitRootLogin Yes
+PasswordAuthentication yes
+PermitEmptyPasswords Yes
+  """
```

Comment:
Another option to make these text blobs a little easier to format would be to use textwrap.

Reply:
Ack